contrib/current-cgroup

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -euo pipefail
+
+if [ "$#" -gt 0 ] && { [ "$1" = "-h" ] || [ "$1" = "--help" ] ; }; then
+  echo "Helper script to find the current cgroup v2 path"
+  echo "Usage: [PID=PROCESS_ID] $0"
+  echo "  If PID is specified as environment variable, the cgroup path of"
+  echo "  the given PID will be queried. Otherwise the PID of the script"
+  echo "  process itself will be used."
+  exit 1
+fi
+
+PID="${PID:-$$}"
+
+if [ -e /sys/fs/cgroup/unified ]; then
+  PREFIX="/sys/fs/cgroup/unified"
+elif [ -e /sys/fs/cgroup/systemd ]; then
+  echo "Error: Traceloop doesn't support cgroup v1" > /dev/stderr
+  exit 1
+else
+  PREFIX="/sys/fs/cgroup"
+fi
+
+CGROUP_PATH="${PREFIX}$(tail -n 1 "/proc/${PID}/cgroup" | cut -d : -f 3-)"
```

Comment:
I think that looking for '0::' is better than `tail -n 1`, in case cgroup2 is not enabled.
```
grep '0::' "/proc/${PID}/cgroup"
```

Reply:
Thanks, updated